toolkit/tools/pkgworker/pkgworker.go

```diff
@@ -82,8 +85,11 @@ func main() {
 	defines[rpm.DistroReleaseVersionDefine] = *distroReleaseVersion
 	defines[rpm.DistroBuildNumberDefine] = *distroBuildNumber
 	defines[rpm.MarinerModuleLdflagsDefine] = "-Wl,-dT,%{_topdir}/BUILD/module_info.ld"
+	if *useCcache {
+		defines[rpm.MarinerCCacheDefine] = "true"
+	}
 
-	builtRPMs, err := buildSRPMInChroot(chrootDir, rpmsDirAbsPath, *workerTar, *srpmFile, *repoFile, *rpmmacrosFile, *outArch, defines, *noCleanup, *runCheck, *packagesToInstall)
+	builtRPMs, err := buildSRPMInChroot(chrootDir, rpmsDirAbsPath, *workerTar, *srpmFile, *repoFile, *rpmmacrosFile, *outArch, defines, *noCleanup, *runCheck, *packagesToInstall, *useCcache)
```

Comment:
[Non blocker] This list of arguments is getting out of hand. As part of this PR, do you think you could also replace it with a struct? Just a kind ask, nothing super important.

Reply:
ran out of time to make this change in this PR, will consider it as a separate change though